DirectProgramming/DPC++FPGA/ReferenceDesigns/merge_sort/README.md

```diff
@@ -159,8 +168,13 @@ You can compile and run this Reference Design in the Eclipse* IDE (in Linux*) an
      ./merge_sort.fpga_emu    (Linux)
      merge_sort.fpga_emu.exe  (Windows)
      ```
+2. Run the sample on the FPGA simulator:
+     ```
+     CL_CONTEXT_MPSIM_DEVICE_INTELFPGA=1 ./merge_sort.fpga_sim     (Linux)
+     CL_CONTEXT_MPSIM_DEVICE_INTELFPGA=1 merge_sort.fpga_sim.exe   (Windows)
```

Comment:
Does this work on windows?  Do you need:
set CL_CONTEXT_MPSIM_DEVICE_INTELFPGA=1
./mer....

Reply:
Assuming you'll modify your sample to use Mark's change, I think that you won't need these CL_CONTEXT_MPSIM_DEVICE_INTELFPGA at all anymore